### DISCUSSION TITLE
Getting space&traffic from Fastly

### DISCUSSION THREAD

**Discussion by paulcarroty** [VSCodium/vscodium#781] [Ideas]
Got the [recommendation](https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/-/issues/35#note_635507661) to apply [Fastly Open Source and Nonprofit Program](https://www.fastly.com/open-source). This will fix Tor issue and help to provide several VSCodium versions in repositories (requested 2 or 3 times) & more.

Ping @stripedpajamas @daiyam

**Comment by daiyam** [comment]
@paulcarroty I don't see any issue.

**Comment by stripedpajamas** [comment]
Same, seems great.

**Comment by paulcarroty** [comment]
No answer, so feel free to ask.

**Comment by daiyam** [comment]
@paulcarroty What the traffic you are getting on your repo? I've read you comment about getting 100k requests per day. Is it still true?

**Reply by paulcarroty** [reply]
The 300 rpm per project limit bypassed a year ago and Gitlab still doesn't provide any traffic stats. So, guess this is best case.
